python/ray/_raylet.pyx

```diff
@@ -109,6 +109,7 @@ include "includes/serialization.pxi"
 include "includes/libcoreworker.pxi"
 include "includes/global_state_accessor.pxi"
 
+OPTIMIZE = __OPTIMIZE__
```

Comment:
What is this used for? Is it some magic constant for cython?

Reply:
I see, it's used to check if the build was optimized, nice. Can you add a comment explaining this?

Reply:
Yup sure!